packages/cfd/src/Containers/cfd-change-password-confirmation.tsx

```diff
@@ -32,6 +33,7 @@ const ChangePasswordConfirmation = ({
                             icon={platform === CFD_PLATFORMS.MT5 ? 'IcMt5OnePassword' : 'IcDxtradeOnePassword'}
                             width='122'
                             height='108'
+                            data_testid='cfd_change_password_confirmation_icon'
```

Comment:
Suggestion based on our testing documentation.
```suggestion
                            data_testid='dt_cfd_change_password_confirmation_icon'
```